app/controllers/shipit/stacks_controller.rb

```diff
@@ -180,7 +180,7 @@ def repo_name
     end
 
     def repository_params
-      params.require(:stack).permit(:repo_owner, :repo_name)
+      params.require(:stack).permit(:repo_owner, :repo_name, :branch, :environment, :deploy_url, :ignore_ci)
```

Comment:
This was updated to prevent the following warning in the logs. Can be easily reverted if necessary/desired.

```
Unpermitted parameters: :branch, :environment, :deploy_url, :ignore_ci
```